examples/example_pump_data_model/default.config.yaml

```diff
@@ -0,0 +1,18 @@
+# Only valid for this module, loads template variables from environment
+model_space: cdfTemplate
+instance_space: cdfTemplateInstances
+source_model_space: ExtendedSourceDataModels
+source_model: ExtendedSourceData
+view_Pump_version: "1"
+view_LiftStation_version: "1"
+data_model_version: "1"
+data_model: "PumpLiftStations"
+clientId: ${IDP_CLIENT_ID}
```

Comment:
I have suggested this before and would like to do it again to avoid lots of duplication: let's move the CI/CD service principal credentials to the global default.config.yaml and make it explicit. E.g. cicd_client_id or similar.

Reply:
Let me do a specific config PR.